src/pages/Attributes.cpp

```diff
@@ -137,6 +151,21 @@ void DevTools::drawNodeAttributes(CCNode* node) {
         }
     }
 
+    if (auto spriteNode = dynamic_cast<CCTextureProtocol*>(node)) {

```

Comment:
This should use `typeinfo_cast`

Reply:
Ok I was just copying the one above with the label protocol